src/main/res/layout/list_item.xml

```diff
@@ -39,13 +40,40 @@
         android:orientation="vertical"
         android:paddingTop="10dp">
 
-        <TextView
-            android:id="@+id/main_text"
-            android:layout_width="fill_parent"
+        <RelativeLayout
+            android:layout_width="match_parent"
             android:layout_height="wrap_content"
+            android:orientation="horizontal"
```

Comment:
A `RelativeLayout` does not have `orientation`

Reply:
removed